### PR TITLE
Oasis pricing table: Add more JS

### PR DESCRIPTION
**Pull request by paulproteus**
This JS makes sure that:

- Clicking the (X) icon in the corner of the modal makes it go
  away without navigating the page, if the page was first-visited
  while not looking at the modal. (Requested by @kentonv)

- Clicking the background will dismiss the modal, too. (Requested
  by @neynah + @paulproteus.)

### CODE REVIEW

**Comment by paulproteus** [comment]
Requesting review from @zarvox and/or @kentonv

File: get.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
 <section id="plan-table">
     <h2>Run Sandstorm anywhere.</h2>
     <ul id class="plans">
-      
+
```
Comment (paulproteus) [inline get.html]: Please ignore this and a few other whitespace-only changes!

File: get.html [hunk 2]
```diff
@@ -182,17 +182,41 @@
...
+    }
+  };
+  var elt = document.getElementById("close-modal");
+  elt.addEventListener("click", dismissOverlay, false);
+
+  /* Make clicking the overlay remove the overlay. */
+  var makeOverlayClicksCloseTheOverlay = function () {
+    var overlays = document.body.getElementsByClassName('overlay');
+    for (var i = 0; i < overlays.length; i++) {
+      overlays[i].addEventListener("click", function(evt) { if (evt.target.classList[0] === "overlay") { if (startedOutAtModal) { window.location = "#"; } else { dismissOverlay(evt); } } } );
```
Comment (zarvox) [inline get.html]: I think it's arguably more correct to write `window.location.href` here?

**Comment by paulproteus** [comment]
Now this passes manual testing. I'll ship it.

**Comment by paulproteus** [comment]
Thanks @zarvox for speedy review!

**Comment by paulproteus** [comment]
@neynah these changes are now live. You might have to wait a while for the CloudFlare cache to expire; I used the script to purge it, but it didn't actually purge, seemingly.

**Pull request merged**